rotkehlchen/api/v1/schemas.py

```diff
@@ -98,6 +98,7 @@
 from rotkehlchen.utils.misc import create_order_by_rules_list, ts_now
 from rotkehlchen.utils.mixins.serializableenum import SerializableEnumMixin
 
+from ...errors.asset import UnknownAsset
```

Comment:
I think this should better be an absolute import

Reply:
yeah, a mistake from my ide during an auto-import